client/shared/src/graphql/schema.ts

```diff
@@ -16319,6 +16319,11 @@ export interface IEventLogsConnection {
 export interface ICodeHostRepositoryConnection {
     __typename: 'CodeHostRepositoryConnection'
 
+    /**
+     * Errors returned from the code host.
+     */
+    codeHostErrors: string | null
```

Comment:
It is a string, so I think it should be named as singular?
```suggestion
    codeHostError: string | null
```

Or make it an array? (I don't know the TS array syntax...)
```suggestion
    codeHostErrors: string[] | null
```